jac-uk/admin
# #2120 Exercise page alignment is breaking after on-screen error messages are displayed

## What's included?
This fix relies on the jac-kit v3.0.37.
The problem was that the scrollIntoView was somehow causing the page content to shift to the left. It's difficult to say exactly why.
The fix involved changing the element being scrolled into view.

Closes #2120 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

- Go to the following page: https://jac-admin-develop--pr2152-hotfix-hotfix-2120-m-6ctuhch9.web.app/exercise/q3r80nVrmVFhB2ML03qq/details/timeline/edit
- Remove one of the dates
- Scroll to the bottom of the page and press the 'Save and Continue' button

The page should scroll back up to display the error summary and the left side of the page should NOT have the display bug as shown in the image below.

<img width="543" alt="Screenshot 2023-09-28 at 14 16 11" src="https://github.com/jac-uk/admin/assets/115651787/52a79561-00b6-4b23-9fda-5f60589d5741">

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
